frontend/packages/dev-console/src/components/import/section/ResourceSection.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import { connectToFlags, FlagsObject } from '@console/internal/reducers/features';
+import {
+  FLAG_KNATIVE_SERVING_SERVICE,
+  ServiceModel,
+  KnativeServingModel,
+} from '@console/knative-plugin';
+import { getBadgeFromType } from '@console/shared';
+import { useAccessReview } from '@console/internal/components/utils';
+import { getActiveNamespace } from '@console/internal/actions/ui';
+import { RadioButtonField } from '../../formik-fields';
+import FormSection from './FormSection';
+import './ResourceSection.scss';
+
+type ServerlessSectionProps = {
+  flags: FlagsObject;
+};
+
+const ResourceSection: React.FC<ServerlessSectionProps> = ({ flags }) => {
+  const radioOptions = [
+    {
+      label: (
+        <div>
+          Kubernetes
+          <br />
+          <div className="odc-resource-section__option-text">
+            This option will produce a Deployment
+          </div>
+        </div>
+      ),
+      value: 'k8s',
+    },
+    {
+      label: (
+        <div>
+          OpenShift
+          <br />
+          <div className="odc-resource-section__option-text">
+            This option will produce a Deployment Config
+          </div>
+        </div>
+      ),
+      value: 'openshift',
+    },
+  ];
```

Comment:
I think `helperText` is a common feature we can leverage here for the radio buttons. I'd like you to do a change here that'll help future users of the `RadioButtonField`.

In `frontend/packages/dev-console/src/components/formik-fields/RadioButtonField.tsx`:
```
        <Radio
          {...field}
          {...props}
          key={option.value}
          id={getFieldId(option.value, 'radiobutton')}
          value={option.value}
          label={
            option.helperText ? (
              <>
                {option.label}
                <div className="odc-radio-button__helper-text">{option.helperText}</div>
              </>
            ) : (
              option.label
            )
          }
          isChecked={field.value === option.value}
          isValid={isValid}
          aria-describedby={`${fieldId}-helper`}
          onChange={(val, event) => {
            field.onChange(event);
          }}
        />
```
and some new styles:
```
.odc-radio-button {
  &__helper-text {
    padding: var(--pf-global--spacer--md) 0;
  }
}
```

You'll need to update the options type (which I'll comment on separately as it has a couple changes needed).

Reply:
done

Reply:
done